packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -49,7 +55,14 @@
             <component
                 :is="errorBoundary"
                 tier="3">
-                <slot name="new-label" />
+                <!-- TODO - we want to translate this within the component using i18n.
+                For now we'll just need to pass down a translated string from the consuming site -->
+                <restaurant-badge
```

Comment:
One thing worth checking with design – I think these are named `Tags` in PIE ([link to the design is here](https://www.figma.com/file/R2rBfzJP0hG0MZorq6FLZ1/%E2%9A%A1%EF%B8%8F%F0%9F%A7%8A--PIE-2.-0-Global---Light-%E2%9A%A1%EF%B8%8F%F0%9F%A7%8A?node-id=742%3A328)). If so, it may be worth updating one way or the other (so either getting them to rename theirs to `badges` or us renaming ours here to reference `Tags `.

Not an urgent one, but would be good to update before we use the badge naming everywhere and it becomes harder to change 😄

Reply:
Ah I see - yes I'm happy to rename to tags :D will do in the next PR! cheers